docs/ml.markdown

```diff
@@ -0,0 +1,183 @@
+---
+title: Machine Learning
+---
+
+# Machine Learning with QuantGov
+
+QuantGov provides a framework for training machine learning estimators
+using a corpus, packaging those estimators, and using those models to
+make predictions about other corpora.
+
+QuantGov builds on the [scikit-learn](http://scikit-learn.org/stable/)
+library, and users training new models should familiarize themselves
+with the models they employ.
+
+## Training a QuantGov Estimator
+
+### Initializing the Estimator
+
+To start a new quantgov estimator, use the following command
+`quantgov start estimator myname` where `myname` is the name you want to
+give to the estimator. This will copy the [estimator
+skeleton](https://github.com/quantgov/estimator) to the folder with that
+name.
+
+### Customize the Vectorizer and Vectorize Trainers
+
```

Comment:
Maybe add one or two sentences that explain what a vectorizer is.

Reply:
Done.